common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -748,7 +748,7 @@ def test_user_can_cancel_the_team_creation(self):
         self.verify_my_team_count(0)
 
 
-@flaky  # TODO fix this, see TNL-3069
+@flaky  # TODO - this is likely not needed after TNL-3069, leaving in to monitor for a week or two
```

Comment:
@jzoldak @cahrens - Should I leave this flaky decorator in place for a week or two to verify the issues have been fixed, or remove it as part of this PR?

I've already gotten two fully successful runs of all tests in TeamPageTest with @flaky(max_runs=10, min_passes=10):
- https://build.testeng.edx.org/view/edx-platform-specific-tests/job/edx-platform-specific-tests/128/
- https://build.testeng.edx.org/view/edx-platform-specific-tests/job/edx-platform-specific-tests/129/